lib/command/definitions.js

```diff
@@ -39,6 +39,7 @@ module.exports = function (genPath, options) {
   const targetFolderPath = options.output && getTestRoot(options.output) || testsPath;
 
   const codecept = new Codecept(config, {});
+  console.log('testpath', testsPath);
```

Comment:
probably use the `output.print`

Reply:
This was a debug message. Thanks for noticing :)